libraries/common/io/console.effekt

```diff
@@ -24,6 +24,11 @@ namespace js {
 
   extern jsNode """
     const readline = require('node:readline');
+    if (process.platform === 'win32') {
+      // Set UTF-8 codepage
+      require('child_process').execSync('chcp 65001');
+      process.stdout.setDefaultEncoding('utf8');
```

Comment:
👀 I _think_ something like this would belong to a `jsNode` block above the definition of `println` in `libraries/common/effekt.effekt` to ensure that the output of `println` itself is UTF-8. Otherwise this line of code only gets interpreted if someone imports `console`...